### ISSUE TITLE
[css-sizing-4] How to interpolate contain-intrinsic-size: auto 10px to 20px

### ISSUE THREAD

**Issue by cbiesinger** [w3c/csswg-drafts#6566]
https://drafts.csswg.org/css-sizing-4/#intrinsic-size-override

let's say we are interpolating from
`contain-intrinsic-size: auto 10px`
to
`contain-intrinsic-size: 20px`
(ie. only one of them has the keyword)

How should it be interpolated? Should the entire thing flip at 50%, or should just "auto" flip at 50% and the length be interpolated linearly?

@flackr

**Comment by flackr** [comment]
My understanding is that given `auto <length>` is a separate kind of value type, it should not be interpolable if the auto value does not match on both sides. I.e. the entire thing (both auto and length) should flip at 50%.

In general, we don't interpolate from auto values to non-auto values, see [width example](https://jsbin.com/pecemiq/edit?html,css,js,output), so if we did interpolate the length and flip the auto you would end up animating from a value that the user never saw if it was using the auto value.

**Comment by tabatkins** [comment]
Yeah, I think that's the right reasoning here. The whole thing should be discrete in this case.
